### DISCUSSION TITLE
script: execute pytest with disected set of pytest extensions (?)

### DISCUSSION THREAD

**Discussion by kloczek** [pytest-dev/pytest#8989] [General]
I'm working on package as rpm and Solaris IPS packages may of python modules.
Each package build procedure described in spec file used on such processes has exact set of byuild dependencies and build is perforemed in lxc zone or non-global zone created to build only one package.

During work on on packagin usulally at the end I'm trying to build package in env where I have installed Everything™️. In such case some additional pytest extension are causing that pytest is failing.
It would be nice to have some tools which will  take list of pytest extensions to try to created set of venvs in which pytest will be executed with disected set of pytest extensions to easy encircle exact pytest extension reportiing some additiona warnings or errors.

Someone maybe already wrote such tool?🤔

**Comment by RonnyPfannschmidt** [comment]
im not aware of such a tool - pytest currently also lacks a mechanism to disable general auto-loading (which creates wel known fragilities)

i hope that eventually we have a mode for pytest, which considers only explicitly requested plug ins and/or has plug-in that require explicit request
